bundles.go

```diff
@@ -75,7 +75,10 @@ func GetEfistub() string {
 }
 
 func NewBundle() *Bundle {
-	esp := GetESP()
+	esp, error := GetESP()
+	if error != nil {
+		panic(error)
```

Comment:
No panics please :)

Reply:
There's another `panic` two lines down.

Reply:
Ohno, I thought I had removed all of them! My misake :) sorry!